templates/base.html

```diff
@@ -95,6 +98,18 @@
                   <a class="nav-link" href="/{{href}}">{{id}}</a>
               </li>
               {% endfor %}
+              {% if organization_bar %}
+              <li class="nav-item dropdown {% if  active_page == 'Organization' %}active{% endif %}">
+                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
This should have a different id to distinguish it from the past-events dropdown

Reply:
Agreed. Will submit in fix momentarily.